docs/config.md

```diff
@@ -188,37 +188,37 @@ documented in `ipfs config profile --help`.
 
 - `flatfs`
 
-  Configures the node to use the flatfs datastore.
+  Configures the node to use the flatfs datastore (the default).
 
-  This is the most battle-tested and reliable datastore, but it's significantly
-  slower than the badger datastore. You should use this datastore if:
+  This is the most battle-tested and reliable datastore. 
+  You should use this datastore if:
 
   - You need a very simple and very reliable datastore and you trust your
     filesystem. This datastore stores each block as a separate file in the
     underlying filesystem so it's unlikely to lose data unless there's an issue
     with the underlying file system.
-  - You need to run garbage collection on a small (<= 10GiB) datastore. The
-    default datastore, badger, can leave several gigabytes of data behind when
-    garbage collecting.
-  - You're concerned about memory usage. In its default configuration, badger can
-    use up to several gigabytes of memory.
+  - You need to run garbage collection in a way that reclaims free space as soon as possible.
+  - You want to minimize memory usage.
+  - You are ok with the default speed of data import (or prefer to use `--nocopy`).
 
   This profile may only be applied when first initializing the node.
 
 
 - `badgerds`
 
-  Configures the node to use the badger datastore.
-
-  This is the fastest datastore. Use this datastore if performance, especially
-  when adding many gigabytes of files, is critical. However:
+  Configures the node to use the experimental badger datastore (warning: uses an outdated badger 1.x).
```

Comment:
```suggestion
  Configures the node to use the experimental badger datastor. Keep in mind that this **uses an outdated badger 1.x**.
```

@lidel is there a specific version number where Badger has become _outdated_? If so, let's add it here in place of `1.x`. If not, just leave the `1.x` in there.

Reply:
There is badger 3.x now, but we want to remove badger in general. 
This note is only to highlight how old the badger in go-ipfs to discourage people from using it even more.